pkg/bootkube/status.go

```diff
@@ -24,7 +25,7 @@ const (
 	doesNotExist = "DoesNotExist"
 )
 
-func WaitUntilPodsRunning(pods []string, timeout time.Duration) error {
+func WaitUntilPodsRunning(pods []string, timeout time.Duration, selfHostedEtcd bool) error {
```

Comment:
can we add a new func to do the etcd migration after calling `WaitUntilPodRunning`? i do not feel this func is a good place.

/cc @aaronlevy might know better than me.